lib/CL/pocl_util.h

```diff
@@ -226,6 +226,8 @@ float half_to_float (uint16_t value);
 
 int bitcode_is_spirv_kernel (const char *bitcode, size_t size);
 
+int bitcode_is_vulkan_spirv (const char *bitcode, size_t size);
```

Comment:
Add a comment here what is exactly meant with "Vulkan SPIR-V" in comparison to the above SPIR-V.

Reply:
ok, added comment, also renamed them to "kernel" and "shader" (`bitcode_is_spirv_execmodel_shader`) as that is the official SPIR-V terminology (and less confusing).